ajax/libs/background-check/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "background-check",
+  "filename": "background-check.min.js",
+  "description": "Automatically switch to a darker or a lighter version of an element depending on the brightness of images behind it.",
+  "homepage": "http://www.kennethcachia.com/background-check/",
+  "keywords": [
+    "background",
+    "check"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kennethcachia/background-check.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kennethcachia/background-check.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "background-check*"
+        ]
+      }
+    ]
+  },
+  "author": "Kenneth Cachia"
```

Comment:
I think you can keep the author's url because it's displayed in the official [package.json ](https://github.com/kennethcachia/background-check/blob/master/package.json#L6)